packages/api/src/services/tileSocket.ts

```diff
@@ -0,0 +1,34 @@
+import { HandleMessage, SocketDefinition } from "../genericTypes/socket";
+import { IdentifySocket } from "./tileSocket/types";
+
+export interface TileFromClientToServer extends HandleMessage {
+  identify: IdentifySocket;
+}
+
+export interface TileFromServerToClient extends HandleMessage {
+  identify: IdentifySocket;
+}
```

Comment:
These are the two important interfaces. If we add a new `FromClientToServer` for example, the BE will complain until the listener is added and the client will get another function it can emit